### PR TITLE
chore: add back to main menu component for login form

### PR DESCRIPTION
**Pull request by zahin-mohammad**
Adds the helper text below the email field, similar to how other components have it listed.

image

image

### CODE REVIEW

File: src/components/BackToHomeHelperText/BackToHomeHelperText.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <div>
+      <span className="tw-text-gray-700">
+        Current environment: &nbsp;
+        <span className="tw-font-semibold">
+          {env === 'prod' ? 'Mainnet' : 'Testnet'}
+        </span>
+        . &nbsp;
+      </span>
+      <Link to="/" className={clsx('tw-text-blue-200', 'hover:tw-underline')}>
```
Comment (mohammadalfaiyazbitgo) [inline src/components/BackToHomeHelperText/BackToHomeHelperText.tsx:19]: is `clsx` needed here? looks like just `'class1 class2'` would work in this case no?

Comment (zahin-mohammad) [reply src/components/BackToHomeHelperText/BackToHomeHelperText.tsx:19]: This PR isn't changing how the component works, just making it re-useable.

Comment (zahin-mohammad) [reply src/components/BackToHomeHelperText/BackToHomeHelperText.tsx:19]: I can take a look if its easy enough to do, but if its not Ill leave that out of this PR

File: src/components/BackToHomeHelperText/BackToHomeHelperText.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <div>
+      <span className="tw-text-gray-700">
+        Current environment: &nbsp;
+        <span className="tw-font-semibold">
+          {env === 'prod' ? 'Mainnet' : 'Testnet'}
+        </span>
+        . &nbsp;
+      </span>
+      <Link to="/" className={'tw-text-blue-200 hover:tw-underline'}>
```
Comment (mohammadalfaiyazbitgo) [inline src/components/BackToHomeHelperText/BackToHomeHelperText.tsx:19]:
```suggestion
      <Link to="/" className="tw-text-blue-200 hover:tw-underline">
```

**Review by mohammadalfaiyazbitgo — APPROVED** [review]
LGTM, but take a look at the nit

**Pull request merged**